Library/Homebrew/sandbox.rb

```diff
@@ -32,7 +32,13 @@ def initialize
   end
 
   def record_log(file)
-    @log = file
+    @logfile = file
+  end
+
+  def self.log_warning_message; <<-EOS.undent
+      We use time to filter sandbox log. Therefore, unrelated logs may be recorded in below.
+
+    EOS
   end
```

Comment:
Yep, if we write it maybe after we print the log then that makes sense to me. Equally if we didn't write it if `HOMEBREW_DEVELOPER` is set or something? Up to you!